app/src/test/java/gov/va/vro/controller/ClaimMetricsControllerTest.java

```diff
@@ -38,377 +22,108 @@
+        // Expect a 200 and a list of size one
+        assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
+        String body = responseEntity.getBody();
+        assertNotNull(body);
+        ExamOrderInfoResponse[] actual = mapper.readValue(body, ExamOrderInfoResponse[].class);
+        assertEquals(1, actual.length);
+        ExamOrderInfoResponse response1 = actual[0];
+        assertNotNull(response1);
+        assertNull(response1.getOrderedAt());
+        assertEquals(response1.getCollectionId(), collectionIDFound);
+        assertEquals(response1.getCreatedAt(), timeFound);
+        assertTrue(slackCalled.get());
     }
-  }
-
-  // Checks if a specific uri results in the expected argument to the service call.
-  private void testExamInfoQueryParamDefaults(String uri, ExamOrderInfoQueryParams expectedParams) {
-    Mockito.reset(service);
-
-    ArgumentCaptor<ExamOrderInfoQueryParams> captor =
-        ArgumentCaptor.forClass(ExamOrderInfoQueryParams.class);
-
-    callRestWithAuthorization(uri);
-
-    Mockito.verify(service).findExamOrderInfo(captor.capture());
-    ExamOrderInfoQueryParams actualParams = captor.getValue();
-
-    assertEquals(expectedParams.getPage(), actualParams.getPage());
-    assertEquals(expectedParams.getSize(), actualParams.getSize());
-    assertEquals(expectedParams.getNotOrdered(), actualParams.getNotOrdered());
-  }
-
-  // Verifies default query parameters results in the expected argument to the service call.
-  @Test
-  void testExamInfoAllQueryParamDefaults() {
-    String uri0 = "/v2/exam-order-info";
-    ExamOrderInfoQueryParams params0 = new ExamOrderInfoQueryParams(0, 10, Boolean.FALSE);
-    testExamInfoQueryParamDefaults(uri0, params0);
-
-    String uri1 = "/v2/exam-order-info?size=15";
-    ExamOrderInfoQueryParams params1 = new ExamOrderInfoQueryParams(0, 15, Boolean.FALSE);
-    testExamInfoQueryParamDefaults(uri1, params1);
-
-    String uri2 = "/v2/exam-order-info?page=1";
-    ExamOrderInfoQueryParams params2 = new ExamOrderInfoQueryParams(1, 10, Boolean.FALSE);
-    testExamInfoQueryParamDefaults(uri2, params2);
-
-    String uri3 = "/v2/exam-order-info?page=1&size=15";
-    ExamOrderInfoQueryParams params3 = new ExamOrderInfoQueryParams(1, 15, Boolean.FALSE);
-    testExamInfoQueryParamDefaults(uri3, params3);
-  }
-
-  @Test
-  void testExamOrderInfoAllInvalidQueryParam() {
-    ResponseEntity<String> re0 = callRestWithAuthorization("/v2/exam-order-info?size=0");
-    assertEquals(HttpStatus.BAD_REQUEST, re0.getStatusCode());
-
-    ResponseEntity<String> re1 = callRestWithAuthorization("/v2/exam-order-info?page=x");
-    assertEquals(HttpStatus.BAD_REQUEST, re1.getStatusCode());
-
-    ResponseEntity<String> re2 = callRestWithAuthorization("/v2/exam-order-info?page=-1");
-    assertEquals(HttpStatus.BAD_REQUEST, re2.getStatusCode());
-  }
-
-  @Test
-  void testExamOrderSlackResponse() throws JsonProcessingException {
```

Comment:
This mockito test was removed and now uses h2 and real service calls in the other file.